### ISSUE TITLE
Smoothing Zones Chunk Won't Run

### ISSUE THREAD

**Issue by dosterharper** [leombastos/2023_AdvPA#74]
I am trying to smooth my cluster map as raster and the chunk will take a long time to run, and then this error message will show:

Error in h(simpleError(msg, call)) :
error in evaluating the argument 'object' in selecting a method for function 'predict': cannot allocate vector of size 1.9 Gb

I have looked it up online and it looks like my computer doesn't have enough memory space to run it?  If that's the case, do I need to shut down applications that are running on my computer or is this because there is too much data?

**Comment by UM004** [comment]
No. I was having similar error and my mistake was using a comma instead of period in IDW code. There a code line for gstate where you write data= .
I had written data = , and I was getting this error, Try changing this and it  might work.

**Comment by leombastos** [comment]
@dosterharper can you show a summary of the data going into the gstat function?

**Comment by dosterharper** [comment]
Thank you both for your help and I finally figured it out.  In joining the data together to create all_v_n for the zones, I was using csv files rather than geojson files.  That wasn't allowing it to run properly.  After I fixed that, I was able to create the zones.
